js/coinsbit.js

```diff
@@ -0,0 +1,1103 @@
+        return this.safeOrder2 ({
+            'id': id,
+            'clientOrderId': undefined, // <<<TODO>>> should it be here?
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'lastTradeTimestamp': undefined,
+            'status': undefined,
+            'symbol': symbol,
+            'type': orderType,
+            'timeInForce': undefined,
+            'side': side,
+            'price': price,
+            'stopPrice': undefined,
+            'average': undefined,
+            'amount': orderSize,
+            'cost': cost,
+            'filled': filledSize,
+            'remaining': remaining,
+            'fee': {
+                'cost': fee,
+                'currency': quoteCurrency,
+                'will_cost': willBeFee,
+            },
+            'trades': undefined,
+            'info': order,
+        }, market);
+    }
+
+    async fetchOrderTrades (id, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {};
+        request['orderId'] = id;
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this.privatePostAccountTrades (this.extend (request, params));
+        // {
+        //     success: true,
+        //     message: '',
+        //     result: {
+        //       offset: '0',
+        //       limit: '50',
+        //       records: {
+        //         time: '1640258718.402',
+        //         fee: '0.01572657',
+        //         price: '3931.64271609',
+        //         amount: '0.002',
+        //         id: '1595921',
+        //         dealOrderId: '8631729321',  //<--- this is different than passed order-id
+        //         role: '2',
+        //         deal: '7.86328543'
+        //       }
+        //     },
+        //     code: '200'
+        // }
+        const result = this.safeValue (response, 'result', {});
+        const order = this.safeValue (result, 'records');
+        if (!order) {
+            throw new OrderNotFound (this.id + ' fetchOrderTrades() could not find any trade data for order-id ' + id);
+        }
+        return this.parseTrade ({ 'id': id, 'order': order }, undefined);
+    }
+
+    codeFromOptions (methodName, params = {}) {
+        const defaultCode = this.safeValue (this.options, 'code', 'BTC');
+        const options = this.safeValue (this.options, methodName, {});
+        const code = this.safeValue (options, 'code', defaultCode);
+        return this.safeValue (params, 'code', code);
+    }
```

Comment:
The above `codeFromOptions` method should be removed, cause it's not used anywhere.